-------------------- Log Start | Fri Dec  8 14:05:27 2023 --------------------
data_path: /home/turbo/printer_data
is_default_data_path: False
config_file: /home/turbo/printer_data/config/moonraker.conf
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: 6f44945189dc46f7a55f40779c59d449
unix_socket_path: /home/turbo/printer_data/comms/moonraker.sock
software_version: v0.8.0-136-g797c623
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/turbo/printer_data/logs/moonraker.log
python_version: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /home/turbo/printer_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager mainsail]
type = web
channel = stable
repo = mainsail-crew/mainsail
path = ~/mainsail

[update_manager moonraker-telegram-bot]
type = git_repo
path = ~/moonraker-telegram-bot
origin = https://github.com/nlef/moonraker-telegram-bot.git
env = ~/moonraker-telegram-bot-env/bin/python
requirements = scripts/requirements.txt
install_script = scripts/install.sh

[update_manager crowsnest]
type = git_repo
path = ~/crowsnest
origin = https://github.com/mainsail-crew/crowsnest.git
install_script = tools/install.sh

[update_manager moonraker-obico]
type = git_repo
path = ~/moonraker-obico
origin = https://github.com/TheSpaghettiDetective/moonraker-obico.git
env = /home/turbo/moonraker-obico-env/bin/python
requirements = requirements.txt
install_script = install.sh
managed_services = 
	moonraker-obico

[update_manager octoeverywhere]
type = git_repo
channel = beta
path = /home/turbo/octoeverywhere
origin = https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere.git
env = /home/turbo/octoeverywhere-env/bin/python
requirements = requirements.txt
install_script = install.sh
managed_services = 
	octoeverywhere

[announcements]
subscriptions = 
	octoeverywhere

[update_manager timelapse]
type = git_repo
primary_branch = main
path = ~/moonraker-timelapse
origin = https://github.com/mainsail-crew/moonraker-timelapse.git
managed_services = klipper moonraker

[timelapse]
output_path = ~/timelapse
frame_path = ~/timelapse/movies

#################################################################
All Configuration Files:
/home/turbo/printer_data/config/moonraker.conf
/home/turbo/printer_data/config/moonraker-obico-update.cfg
/home/turbo/printer_data/config/octoeverywhere-system.cfg
#################################################################
2023-12-08 15:05:27,918 [confighelper.py:read_file()] - Configuration File '/home/turbo/printer_data/config/moonraker.conf' parsed, total size: 2606 B
2023-12-08 15:05:27,919 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /home/turbo/printer_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager mainsail]
type = web
channel = stable
repo = mainsail-crew/mainsail
path = ~/mainsail

[update_manager moonraker-telegram-bot]
type = git_repo
path = ~/moonraker-telegram-bot
origin = https://github.com/nlef/moonraker-telegram-bot.git
env = ~/moonraker-telegram-bot-env/bin/python
requirements = scripts/requirements.txt
install_script = scripts/install.sh

[update_manager crowsnest]
type = git_repo
path = ~/crowsnest
origin = https://github.com/mainsail-crew/crowsnest.git
install_script = tools/install.sh

[update_manager moonraker-obico]
type = git_repo
path = ~/moonraker-obico
origin = https://github.com/TheSpaghettiDetective/moonraker-obico.git
env = /home/turbo/moonraker-obico-env/bin/python
requirements = requirements.txt
install_script = install.sh
managed_services = 
	moonraker-obico

[update_manager octoeverywhere]
type = git_repo
channel = beta
path = /home/turbo/octoeverywhere
origin = https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere.git
env = /home/turbo/octoeverywhere-env/bin/python
requirements = requirements.txt
install_script = install.sh
managed_services = 
	octoeverywhere

[announcements]
subscriptions = 
	octoeverywhere

[update_manager timelapse]
type = git_repo
primary_branch = main
path = ~/moonraker-timelapse
origin = https://github.com/mainsail-crew/moonraker-timelapse.git
managed_services = klipper moonraker

[timelapse]
output_path = ~/timelapse
frame_path = ~/timelapse/movies

#################################################################
All Configuration Files:
/home/turbo/printer_data/config/moonraker.conf
/home/turbo/printer_data/config/moonraker-obico-update.cfg
/home/turbo/printer_data/config/octoeverywhere-system.cfg
#################################################################
2023-12-08 15:05:27,946 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/logs/rollover
2023-12-08 15:05:27,946 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.logs.rollover
2023-12-08 15:05:27,946 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/info
2023-12-08 15:05:27,947 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.info
2023-12-08 15:05:27,947 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/config
2023-12-08 15:05:27,947 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.config
2023-12-08 15:05:27,947 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/restart
2023-12-08 15:05:27,958 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.restart
2023-12-08 15:05:28,062 [server.py:load_component()] - Component (dbus_manager) loaded
2023-12-08 15:05:28,159 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 17
2023-12-08 15:05:28,176 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/database/list
2023-12-08 15:05:28,176 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.database.list
2023-12-08 15:05:28,176 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET POST DELETE) /server/database/item
2023-12-08 15:05:28,177 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.database.get_item, server.database.post_item, server.database.delete_item
2023-12-08 15:05:28,177 [server.py:load_component()] - Component (database) loaded
2023-12-08 15:05:28,228 [file_manager.py:__init__()] - Using File System Observer: inotify
2023-12-08 15:05:28,254 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/files/list
2023-12-08 15:05:28,257 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.list
2023-12-08 15:05:28,257 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/files/metadata
2023-12-08 15:05:28,258 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.metadata
2023-12-08 15:05:28,259 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/files/metascan
2023-12-08 15:05:28,259 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.metascan
2023-12-08 15:05:28,259 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/files/thumbnails
2023-12-08 15:05:28,259 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.thumbnails
2023-12-08 15:05:28,259 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/files/roots
2023-12-08 15:05:28,259 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.roots
2023-12-08 15:05:28,259 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET POST DELETE) /server/files/directory
2023-12-08 15:05:28,259 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.get_directory, server.files.post_directory, server.files.delete_directory
2023-12-08 15:05:28,259 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/files/move
2023-12-08 15:05:28,259 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.move
2023-12-08 15:05:28,260 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/files/copy
2023-12-08 15:05:28,263 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.copy
2023-12-08 15:05:28,263 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/files/zip
2023-12-08 15:05:28,264 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.zip
2023-12-08 15:05:28,264 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.files.delete_file
2023-12-08 15:05:28,291 [server.py:load_component()] - Component (file_manager) loaded
2023-12-08 15:05:28,296 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /printer/print/pause
2023-12-08 15:05:28,297 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.print.pause
2023-12-08 15:05:28,297 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /printer/print/resume
2023-12-08 15:05:28,297 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.print.resume
2023-12-08 15:05:28,297 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /printer/print/cancel
2023-12-08 15:05:28,297 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.print.cancel
2023-12-08 15:05:28,297 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /printer/print/start
2023-12-08 15:05:28,297 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.print.start
2023-12-08 15:05:28,297 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /printer/restart
2023-12-08 15:05:28,297 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.restart
2023-12-08 15:05:28,298 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /printer/firmware_restart
2023-12-08 15:05:28,298 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.firmware_restart
2023-12-08 15:05:28,298 [server.py:load_component()] - Component (klippy_apis) loaded
2023-12-08 15:05:28,576 [server.py:load_component()] - Component (secrets) loaded
2023-12-08 15:05:28,584 [server.py:load_component()] - Component (template) loaded
2023-12-08 15:05:28,606 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: sys.version_info(major=3, minor=10, micro=12, releaselevel='final', serial=0)
  version_string: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]

***cpu_info***
  cpu_count: 2
  bits: 64bit
  processor: x86_64
  cpu_desc: Intel(R) Core(TM)2 Duo CPU     E8400  @ 3.00GHz
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 5892288
  memory_units: kB

***sd_info***

***distribution***
  name: Ubuntu 22.04.3 LTS
  id: ubuntu
  version: 22.04
  version_parts: {'major': '22', 'minor': '04', 'build_number': ''}
  like: debian
  codename: jammy
  release_info: {}

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
2023-12-08 15:05:28,609 [server.py:load_component()] - Component (shell_command) loaded
2023-12-08 15:05:28,609 [machine.py:__init__()] - Using System Provider: systemd_dbus
2023-12-08 15:05:28,609 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/reboot
2023-12-08 15:05:28,609 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.reboot
2023-12-08 15:05:28,609 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/shutdown
2023-12-08 15:05:28,610 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.shutdown
2023-12-08 15:05:28,610 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/services/restart
2023-12-08 15:05:28,610 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.services.restart
2023-12-08 15:05:28,610 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/services/stop
2023-12-08 15:05:28,610 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.services.stop
2023-12-08 15:05:28,610 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/services/start
2023-12-08 15:05:28,610 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.services.start
2023-12-08 15:05:28,610 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /machine/system_info
2023-12-08 15:05:28,610 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.system_info
2023-12-08 15:05:28,610 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /machine/sudo/info
2023-12-08 15:05:28,611 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.sudo.info
2023-12-08 15:05:28,611 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/sudo/password
2023-12-08 15:05:28,617 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.sudo.password
2023-12-08 15:05:28,623 [server.py:load_component()] - Component (machine) loaded
2023-12-08 15:05:28,627 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/temperature_store
2023-12-08 15:05:28,628 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.temperature_store
2023-12-08 15:05:28,628 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/gcode_store
2023-12-08 15:05:28,628 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.gcode_store
2023-12-08 15:05:28,628 [server.py:load_component()] - Component (data_store) loaded
2023-12-08 15:05:28,633 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2023-12-08 15:05:28,633 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /machine/proc_stats
2023-12-08 15:05:28,633 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.proc_stats
2023-12-08 15:05:28,633 [server.py:load_component()] - Component (proc_stats) loaded
2023-12-08 15:05:28,639 [server.py:load_component()] - Component (job_state) loaded
2023-12-08 15:05:28,646 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST DELETE) /server/job_queue/job
2023-12-08 15:05:28,649 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.job_queue.post_job, server.job_queue.delete_job
2023-12-08 15:05:28,649 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/job_queue/pause
2023-12-08 15:05:28,649 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.job_queue.pause
2023-12-08 15:05:28,649 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/job_queue/start
2023-12-08 15:05:28,649 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.job_queue.start
2023-12-08 15:05:28,649 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/job_queue/status
2023-12-08 15:05:28,649 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.job_queue.status
2023-12-08 15:05:28,649 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/job_queue/jump
2023-12-08 15:05:28,650 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.job_queue.jump
2023-12-08 15:05:28,650 [server.py:load_component()] - Component (job_queue) loaded
2023-12-08 15:05:28,669 [server.py:load_component()] - Component (http_client) loaded
2023-12-08 15:05:28,675 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/announcements/list
2023-12-08 15:05:28,675 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.announcements.list
2023-12-08 15:05:28,675 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/announcements/dismiss
2023-12-08 15:05:28,676 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.announcements.dismiss
2023-12-08 15:05:28,676 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/announcements/update
2023-12-08 15:05:28,676 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.announcements.update
2023-12-08 15:05:28,676 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST DELETE) /server/announcements/feed
2023-12-08 15:05:28,676 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.announcements.post_feed, server.announcements.delete_feed
2023-12-08 15:05:28,676 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/announcements/feeds
2023-12-08 15:05:28,676 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.announcements.feeds
2023-12-08 15:05:28,676 [server.py:load_component()] - Component (announcements) loaded
2023-12-08 15:05:28,695 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/webcams/list
2023-12-08 15:05:28,695 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.webcams.list
2023-12-08 15:05:28,695 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET POST DELETE) /server/webcams/item
2023-12-08 15:05:28,696 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.webcams.get_item, server.webcams.post_item, server.webcams.delete_item
2023-12-08 15:05:28,696 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/webcams/test
2023-12-08 15:05:28,696 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.webcams.test
2023-12-08 15:05:28,696 [server.py:load_component()] - Component (webcam) loaded
2023-12-08 15:05:28,699 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: connection.send_event
2023-12-08 15:05:28,700 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/extensions/list
2023-12-08 15:05:28,700 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.extensions.list
2023-12-08 15:05:28,700 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/extensions/request
2023-12-08 15:05:28,700 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.extensions.request
2023-12-08 15:05:28,700 [server.py:load_component()] - Component (extensions) loaded
2023-12-08 15:05:28,746 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
10.0.0.0/8
127.0.0.0/8
169.254.0.0/16
172.16.0.0/12
192.168.0.0/16
fe80::/10
::1/128
CORS Domains:
.*\.lan
.*\.local
.*://localhost
.*://localhost:.*
.*://my\.mainsail\.xyz
.*://app\.fluidd\.xyz
2023-12-08 15:05:28,746 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /access/login
2023-12-08 15:05:28,747 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.login
2023-12-08 15:05:28,747 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /access/logout
2023-12-08 15:05:28,748 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.logout
2023-12-08 15:05:28,748 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /access/refresh_jwt
2023-12-08 15:05:28,748 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.refresh_jwt
2023-12-08 15:05:28,748 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET POST DELETE) /access/user
2023-12-08 15:05:28,748 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.get_user, access.post_user, access.delete_user
2023-12-08 15:05:28,748 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /access/users/list
2023-12-08 15:05:28,748 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.users.list
2023-12-08 15:05:28,748 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /access/user/password
2023-12-08 15:05:28,748 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.user.password
2023-12-08 15:05:28,748 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET POST) /access/api_key
2023-12-08 15:05:28,748 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.get_api_key, access.post_api_key
2023-12-08 15:05:28,749 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /access/oneshot_token
2023-12-08 15:05:28,749 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.oneshot_token
2023-12-08 15:05:28,749 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /access/info
2023-12-08 15:05:28,749 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: access.info
2023-12-08 15:05:28,749 [server.py:load_component()] - Component (authorization) loaded
2023-12-08 15:05:28,766 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/timelapse/render
2023-12-08 15:05:28,766 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.timelapse.render
2023-12-08 15:05:28,766 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/timelapse/saveframes
2023-12-08 15:05:28,767 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.timelapse.saveframes
2023-12-08 15:05:28,767 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET POST) /machine/timelapse/settings
2023-12-08 15:05:28,768 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.timelapse.get_settings, machine.timelapse.post_settings
2023-12-08 15:05:28,768 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /machine/timelapse/lastframeinfo
2023-12-08 15:05:28,768 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.timelapse.lastframeinfo
2023-12-08 15:05:28,768 [server.py:load_component()] - Component (timelapse) loaded
2023-12-08 15:05:28,773 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET DELETE) /server/history/job
2023-12-08 15:05:28,774 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.history.get_job, server.history.delete_job
2023-12-08 15:05:28,774 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/history/list
2023-12-08 15:05:28,774 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.history.list
2023-12-08 15:05:28,774 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /server/history/totals
2023-12-08 15:05:28,774 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.history.totals
2023-12-08 15:05:28,774 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /server/history/reset_totals
2023-12-08 15:05:28,774 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: server.history.reset_totals
2023-12-08 15:05:28,784 [server.py:load_component()] - Component (history) loaded
2023-12-08 15:05:28,787 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /api/version
2023-12-08 15:05:28,787 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /api/server
2023-12-08 15:05:28,787 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /api/login
2023-12-08 15:05:28,787 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /api/currentuser
2023-12-08 15:05:28,787 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /api/settings
2023-12-08 15:05:28,787 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /api/job
2023-12-08 15:05:28,788 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /api/printer
2023-12-08 15:05:28,788 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /api/printer/command
2023-12-08 15:05:28,788 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /api/printerprofiles
2023-12-08 15:05:28,788 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /api/files/moonraker/(?P<relative_path>.+)
2023-12-08 15:05:28,788 [server.py:load_component()] - Component (octoprint_compat) loaded
2023-12-08 15:05:28,873 [base_deploy.py:log_info()] - Git Repo moonraker: Detected virtualenv: /home/turbo/moonraker-env
2023-12-08 15:05:28,882 [base_deploy.py:log_info()] - Git Repo klipper: Detected virtualenv: /home/turbo/klippy-env
2023-12-08 15:05:28,883 [base_deploy.py:log_info()] - Git Repo moonraker-telegram-bot: Option 'env' is deprecated, use 'virtualenv' instead.
2023-12-08 15:05:28,884 [base_deploy.py:log_info()] - Git Repo moonraker-telegram-bot: Detected virtualenv: /home/turbo/moonraker-telegram-bot-env
2023-12-08 15:05:28,890 [base_deploy.py:log_info()] - Git Repo moonraker-obico: Option 'env' is deprecated, use 'virtualenv' instead.
2023-12-08 15:05:28,891 [base_deploy.py:log_info()] - Git Repo moonraker-obico: Detected virtualenv: /home/turbo/moonraker-obico-env
2023-12-08 15:05:28,894 [base_deploy.py:log_info()] - Git Repo octoeverywhere: Option 'env' is deprecated, use 'virtualenv' instead.
2023-12-08 15:05:28,895 [base_deploy.py:log_info()] - Git Repo octoeverywhere: Detected virtualenv: /home/turbo/octoeverywhere-env
2023-12-08 15:05:28,897 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/moonraker
2023-12-08 15:05:28,898 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.moonraker
2023-12-08 15:05:28,898 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/klipper
2023-12-08 15:05:28,898 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.klipper
2023-12-08 15:05:28,898 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/system
2023-12-08 15:05:28,898 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.system
2023-12-08 15:05:28,898 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/client
2023-12-08 15:05:28,898 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.client
2023-12-08 15:05:28,898 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/full
2023-12-08 15:05:28,898 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.full
2023-12-08 15:05:28,899 [app.py:register_local_handler()] - Registering HTTP Endpoint: (GET) /machine/update/status
2023-12-08 15:05:28,899 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.status
2023-12-08 15:05:28,899 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/refresh
2023-12-08 15:05:28,899 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.refresh
2023-12-08 15:05:28,899 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/recover
2023-12-08 15:05:28,899 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.recover
2023-12-08 15:05:28,899 [app.py:register_local_handler()] - Registering HTTP Endpoint: (POST) /machine/update/rollback
2023-12-08 15:05:28,899 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: machine.update.rollback
2023-12-08 15:05:28,899 [server.py:load_component()] - Component (update_manager) loaded
2023-12-08 15:05:28,911 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2023-12-08 15:05:28,922 [server.py:_initialize_component()] - Performing Component Post Init: [file_manager]
2023-12-08 15:05:29,211 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2023-12-08 15:05:29,223 [machine.py:validation_init()] - Installation version in database up to date
2023-12-08 15:05:29,628 [machine.py:check_virt_status()] - No Virtualization Detected
2023-12-08 15:05:29,651 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.223
2023-12-08 15:05:29,653 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://192.168.1.223
2023-12-08 15:05:29,744 [server.py:add_log_rollover_item()] - 
Systemd unit moonraker.service:
unit_name: moonraker.service
is_default: True
manager: systemd
Properties:
**Requires=['sysinit.target', '-.mount', 'system.slice', 'network-online.target']
**After=['sysinit.target', '-.mount', 'systemd-journald.socket', 'basic.target', 'system.slice', 'network-online.target']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/turbo/printer_data/systemd/moonraker.env
**ExecStart=/home/turbo/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/turbo/moonraker
**FragmentPath=/etc/systemd/system/moonraker.service
**Description=API Server for Klipper SV1
**User=turbo
2023-12-08 15:05:29,745 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2023-12-08 15:05:29,745 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2023-12-08 15:05:29,752 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2023-12-08 15:05:29,752 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://192.168.1.223
2023-12-08 15:05:29,753 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2023-12-08 15:05:29,753 [server.py:add_log_rollover_item()] - 
Systemd unit klipper.service:
unit_name: klipper.service
is_default: True
manager: systemd
Properties:
**Requires=['system.slice', 'sysinit.target', '-.mount']
**After=['network-online.target', 'system.slice', 'sysinit.target', 'systemd-journald.socket', 'basic.target', '-.mount']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/turbo/printer_data/systemd/klipper.env
**ExecStart=/home/turbo/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/turbo/klipper
**FragmentPath=/etc/systemd/system/klipper.service
**Description=Klipper 3D Printer Firmware SV1
**User=turbo
2023-12-08 15:05:29,753 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2023-12-08 15:05:29,753 [server.py:_initialize_component()] - Performing Component Post Init: [timelapse]
2023-12-08 15:05:29,753 [timelapse.py:parseWebcamConfig()] - snapshoturl: http://localhost/webcam/?action=snapshot, Flip V/H: False/False, rotation: 0
2023-12-08 15:05:29,753 [server.py:_initialize_component()] - Performing Component Post Init: [update_manager]
2023-12-08 15:05:29,948 [base_deploy.py:log_info()] - PackageDeploy: PackageKit Provider Configured
2023-12-08 15:05:29,950 [base_deploy.py:log_info()] - Git Repo moonraker: Stored pip version: 23.0
2023-12-08 15:05:30,000 [base_deploy.py:log_info()] - Git Repo octoeverywhere: Stored pip version: 23.0
2023-12-08 15:05:30,037 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/turbo/printer_data/comms/moonraker.sock'
2023-12-08 15:05:30,039 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: DELL
2023-12-08 15:05:30,040 [app.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2023-12-08 15:05:30,299 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2023-12-08 15:05:30,360 [server.py:add_log_rollover_item()] - 
Systemd unit klipper.service:
unit_name: klipper.service
is_default: True
manager: systemd
Properties:
**Requires=['system.slice', 'sysinit.target', '-.mount']
**After=['-.mount', 'system.slice', 'basic.target', 'network-online.target', 'systemd-journald.socket', 'sysinit.target']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/turbo/printer_data/systemd/klipper.env
**ExecStart=/home/turbo/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/turbo/klipper
**FragmentPath=/etc/systemd/system/klipper.service
**Description=Klipper 3D Printer Firmware SV1
**User=turbo
2023-12-08 15:05:30,611 [server.py:add_log_rollover_item()] - Klipper Version: v0.11.0-255-g6f7c7436
2023-12-08 15:05:30,614 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/info
2023-12-08 15:05:30,614 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.info
2023-12-08 15:05:30,614 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/emergency_stop
2023-12-08 15:05:30,615 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.emergency_stop
2023-12-08 15:05:30,615 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/gcode/help
2023-12-08 15:05:30,615 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.gcode.help
2023-12-08 15:05:30,615 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/gcode/script
2023-12-08 15:05:30,615 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.gcode.script
2023-12-08 15:05:30,615 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/gcode/restart
2023-12-08 15:05:30,615 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.gcode.restart
2023-12-08 15:05:30,615 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/gcode/firmware_restart
2023-12-08 15:05:30,615 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.gcode.firmware_restart
2023-12-08 15:05:30,615 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/objects/list
2023-12-08 15:05:30,616 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.objects.list
2023-12-08 15:05:30,616 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/objects/query
2023-12-08 15:05:30,616 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.objects.query
2023-12-08 15:05:30,616 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/objects/subscribe
2023-12-08 15:05:30,616 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.objects.subscribe
2023-12-08 15:05:30,616 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/pause_resume/cancel
2023-12-08 15:05:30,616 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.pause_resume.cancel
2023-12-08 15:05:30,616 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/pause_resume/pause
2023-12-08 15:05:30,616 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.pause_resume.pause
2023-12-08 15:05:30,616 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/pause_resume/resume
2023-12-08 15:05:30,616 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.pause_resume.resume
2023-12-08 15:05:30,617 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/motion_report/dump_stepper
2023-12-08 15:05:30,617 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.motion_report.dump_stepper
2023-12-08 15:05:30,617 [app.py:register_remote_handler()] - Registering HTTP endpoint: (GET POST) /printer/query_endstops/status
2023-12-08 15:05:30,617 [websockets.py:register_api_handler()] - Registering Websocket JSON-RPC methods: printer.query_endstops.status
2023-12-08 15:05:33,688 [app.py:log_request()] - 404 GET /api/ (127.0.0.1) [No User] 6.64ms
2023-12-08 15:05:37,240 [base_deploy.py:log_info()] - PackageKit: Detected 2 package updates:
kpartx
kpartx-boot
2023-12-08 15:05:37,261 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker status --porcelain -b) successfully finished
2023-12-08 15:05:37,279 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker branch --list --no-color) successfully finished
2023-12-08 15:05:37,287 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker remote get-url origin) successfully finished
2023-12-08 15:05:49,692 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d0b3c550>, <Task finished name='Task-239' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:05:55,279 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d00fdab0>, <Task finished name='Task-337' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:05:56,769 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': Timeout while connecting
2023-12-08 15:05:57,381 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker fetch origin --prune --progress) exited with return code 128
2023-12-08 15:05:57,385 [shell_command.py:_check_proc_success()] - Command (find .git/objects/ -type f -empty | xargs rm) exited with return code 1
2023-12-08 15:05:57,887 [git_deploy.py:_repair_loose_objects()] - Attempt to repair loose objects failed, hard recovery is required
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1196, in _repair_loose_objects
    await self.cmd_helper.run_cmd_with_response(
  File "/home/turbo/moonraker/moonraker/components/shell_command.py", line 276, in run_with_response
    raise ShellCommandError(
moonraker.components.shell_command.ShellCommandError: Error running shell command: 'find .git/objects/ -type f -empty | xargs rm'
2023-12-08 15:05:57,887 [git_deploy.py:refresh_repo_state()] - Git Repo moonraker: Initialization failure
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:05:57,887 [git_deploy.py:refresh()] - Error Refreshing git state
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 57, in refresh
    await self._update_repo_state()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 63, in _update_repo_state
    await self.repo.refresh_repo_state(need_fetch=need_fetch)
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:05:57,919 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/klipper status --porcelain -b) successfully finished
2023-12-08 15:05:57,927 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/klipper branch --list --no-color) successfully finished
2023-12-08 15:05:57,933 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/klipper remote get-url origin) successfully finished
2023-12-08 15:05:59,258 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d012dea0>, <Task finished name='Task-414' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:04,259 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d012e320>, <Task finished name='Task-460' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:05,400 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d0b3c3a0>, <Task finished name='Task-505' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:10,404 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d00fda20>, <Task finished name='Task-545' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:10,406 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/klipper fetch origin --prune --progress) exited with return code 128
2023-12-08 15:06:10,409 [shell_command.py:_check_proc_success()] - Command (find .git/objects/ -type f -empty | xargs rm) exited with return code 1
2023-12-08 15:06:10,909 [git_deploy.py:_repair_loose_objects()] - Attempt to repair loose objects failed, hard recovery is required
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1196, in _repair_loose_objects
    await self.cmd_helper.run_cmd_with_response(
  File "/home/turbo/moonraker/moonraker/components/shell_command.py", line 276, in run_with_response
    raise ShellCommandError(
moonraker.components.shell_command.ShellCommandError: Error running shell command: 'find .git/objects/ -type f -empty | xargs rm'
2023-12-08 15:06:10,909 [git_deploy.py:refresh_repo_state()] - Git Repo klipper: Initialization failure
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:10,909 [git_deploy.py:refresh()] - Error Refreshing git state
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 57, in refresh
    await self._update_repo_state()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 63, in _update_repo_state
    await self.repo.refresh_repo_state(need_fetch=need_fetch)
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:19,173 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d012e050>, <Task finished name='Task-630' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:19,173 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d012e440>, <Task finished name='Task-671' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:19,173 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d00fdc60>, <Task finished name='Task-727' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:19,173 [ioloop.py:_run_callback()] - Exception in callback functools.partial(<function _HTTPConnection.__init__.<locals>.<lambda> at 0x7f96d012dbd0>, <Task finished name='Task-708' coro=<_HTTPConnection.run() done, defined at /home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py:290> exception=gaierror(-3, 'Temporary failure in name resolution')>)
Traceback (most recent call last):
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/ioloop.py", line 740, in _run_callback
    ret = callback()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 287, in <lambda>
    gen.convert_yielded(self.run()), lambda f: f.result()
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/simple_httpclient.py", line 340, in run
    stream = await self.tcp_client.connect(
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/tcpclient.py", line 265, in connect
    addrinfo = await self.resolver.resolve(host, port, af)
  File "/home/turbo/moonraker-env/lib/python3.10/site-packages/tornado/netutil.py", line 433, in resolve
    for fam, _, _, _, address in await asyncio.get_running_loop().getaddrinfo(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 863, in getaddrinfo
    return await self.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:19,180 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:19,684 [base_deploy.py:log_info()] - Web Client mainsail: Github Request Error - [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:19,684 [web_deploy.py:_log_client_info()] - Web Client mainsail Detected:
Repo: mainsail-crew/mainsail
Channel: stable
Path: /home/turbo/mainsail
Local Version: v2.6.2
Remote Version: v2.8.0
Valid: True
Fallback Client Detected: False
Pre-release: False
Download Url: https://github.com/mainsail-crew/mainsail/releases/download/v2.8.0/mainsail.zip
Download Size: 2147663
Content Type: application/zip
Rollback Version: v2.6.2
Rollback Repo: mainsail-crew/mainsail
2023-12-08 15:06:19,701 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-telegram-bot status --porcelain -b) successfully finished
2023-12-08 15:06:19,713 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-telegram-bot branch --list --no-color) successfully finished
2023-12-08 15:06:19,723 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-telegram-bot remote get-url origin) successfully finished
2023-12-08 15:06:19,744 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-telegram-bot fetch origin --prune --progress) exited with return code 128
2023-12-08 15:06:19,749 [shell_command.py:_check_proc_success()] - Command (find .git/objects/ -type f -empty | xargs rm) exited with return code 1
2023-12-08 15:06:19,809 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.223
2023-12-08 15:06:20,250 [git_deploy.py:_repair_loose_objects()] - Attempt to repair loose objects failed, hard recovery is required
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1196, in _repair_loose_objects
    await self.cmd_helper.run_cmd_with_response(
  File "/home/turbo/moonraker/moonraker/components/shell_command.py", line 276, in run_with_response
    raise ShellCommandError(
moonraker.components.shell_command.ShellCommandError: Error running shell command: 'find .git/objects/ -type f -empty | xargs rm'
2023-12-08 15:06:20,250 [git_deploy.py:refresh_repo_state()] - Git Repo moonraker-telegram-bot: Initialization failure
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:20,250 [git_deploy.py:refresh()] - Error Refreshing git state
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 57, in refresh
    await self._update_repo_state()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 63, in _update_repo_state
    await self.repo.refresh_repo_state(need_fetch=need_fetch)
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:20,270 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/crowsnest status --porcelain -b) successfully finished
2023-12-08 15:06:20,276 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/crowsnest branch --list --no-color) successfully finished
2023-12-08 15:06:20,281 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/crowsnest remote get-url origin) successfully finished
2023-12-08 15:06:20,295 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/crowsnest fetch origin --prune --progress) exited with return code 128
2023-12-08 15:06:20,297 [shell_command.py:_check_proc_success()] - Command (find .git/objects/ -type f -empty | xargs rm) exited with return code 1
2023-12-08 15:06:20,798 [git_deploy.py:_repair_loose_objects()] - Attempt to repair loose objects failed, hard recovery is required
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1196, in _repair_loose_objects
    await self.cmd_helper.run_cmd_with_response(
  File "/home/turbo/moonraker/moonraker/components/shell_command.py", line 276, in run_with_response
    raise ShellCommandError(
moonraker.components.shell_command.ShellCommandError: Error running shell command: 'find .git/objects/ -type f -empty | xargs rm'
2023-12-08 15:06:20,798 [git_deploy.py:refresh_repo_state()] - Git Repo crowsnest: Initialization failure
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:20,798 [git_deploy.py:refresh()] - Error Refreshing git state
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 57, in refresh
    await self._update_repo_state()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 63, in _update_repo_state
    await self.repo.refresh_repo_state(need_fetch=need_fetch)
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:20,852 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-obico status --porcelain -b) successfully finished
2023-12-08 15:06:20,858 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-obico branch --list --no-color) successfully finished
2023-12-08 15:06:20,863 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-obico remote get-url origin) successfully finished
2023-12-08 15:06:20,877 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-obico fetch origin --prune --progress) exited with return code 128
2023-12-08 15:06:20,880 [shell_command.py:_check_proc_success()] - Command (find .git/objects/ -type f -empty | xargs rm) exited with return code 1
2023-12-08 15:06:21,195 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 15:06:21,380 [git_deploy.py:_repair_loose_objects()] - Attempt to repair loose objects failed, hard recovery is required
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1196, in _repair_loose_objects
    await self.cmd_helper.run_cmd_with_response(
  File "/home/turbo/moonraker/moonraker/components/shell_command.py", line 276, in run_with_response
    raise ShellCommandError(
moonraker.components.shell_command.ShellCommandError: Error running shell command: 'find .git/objects/ -type f -empty | xargs rm'
2023-12-08 15:06:21,380 [git_deploy.py:refresh_repo_state()] - Git Repo moonraker-obico: Initialization failure
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:21,380 [git_deploy.py:refresh()] - Error Refreshing git state
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 57, in refresh
    await self._update_repo_state()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 63, in _update_repo_state
    await self.repo.refresh_repo_state(need_fetch=need_fetch)
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:21,420 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/octoeverywhere status --porcelain -b) successfully finished
2023-12-08 15:06:21,426 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/octoeverywhere branch --list --no-color) successfully finished
2023-12-08 15:06:21,432 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/octoeverywhere remote get-url origin) successfully finished
2023-12-08 15:06:21,446 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/octoeverywhere fetch origin --prune --progress) exited with return code 128
2023-12-08 15:06:21,448 [shell_command.py:_check_proc_success()] - Command (find .git/objects/ -type f -empty | xargs rm) exited with return code 1
2023-12-08 15:06:21,949 [git_deploy.py:_repair_loose_objects()] - Attempt to repair loose objects failed, hard recovery is required
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1196, in _repair_loose_objects
    await self.cmd_helper.run_cmd_with_response(
  File "/home/turbo/moonraker/moonraker/components/shell_command.py", line 276, in run_with_response
    raise ShellCommandError(
moonraker.components.shell_command.ShellCommandError: Error running shell command: 'find .git/objects/ -type f -empty | xargs rm'
2023-12-08 15:06:21,949 [git_deploy.py:refresh_repo_state()] - Git Repo octoeverywhere: Initialization failure
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:21,949 [git_deploy.py:refresh()] - Error Refreshing git state
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 57, in refresh
    await self._update_repo_state()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 63, in _update_repo_state
    await self.repo.refresh_repo_state(need_fetch=need_fetch)
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:21,961 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-timelapse status --porcelain -b) successfully finished
2023-12-08 15:06:21,967 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-timelapse branch --list --no-color) successfully finished
2023-12-08 15:06:21,972 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-timelapse remote get-url origin) successfully finished
2023-12-08 15:06:29,814 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.223
2023-12-08 15:06:41,885 [shell_command.py:_check_proc_success()] - Command (git -C /home/turbo/moonraker-timelapse fetch origin --prune --progress) exited with return code 128
2023-12-08 15:06:41,890 [shell_command.py:_check_proc_success()] - Command (find .git/objects/ -type f -empty | xargs rm) exited with return code 1
2023-12-08 15:06:42,390 [git_deploy.py:_repair_loose_objects()] - Attempt to repair loose objects failed, hard recovery is required
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1196, in _repair_loose_objects
    await self.cmd_helper.run_cmd_with_response(
  File "/home/turbo/moonraker/moonraker/components/shell_command.py", line 276, in run_with_response
    raise ShellCommandError(
moonraker.components.shell_command.ShellCommandError: Error running shell command: 'find .git/objects/ -type f -empty | xargs rm'
2023-12-08 15:06:42,390 [git_deploy.py:refresh_repo_state()] - Git Repo timelapse: Initialization failure
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:42,390 [git_deploy.py:refresh()] - Error Refreshing git state
Traceback (most recent call last):
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 57, in refresh
    await self._update_repo_state()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 63, in _update_repo_state
    await self.repo.refresh_repo_state(need_fetch=need_fetch)
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 388, in refresh_repo_state
    await self.fetch()
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 796, in fetch
    await self._run_git_cmd_async(
  File "/home/turbo/moonraker/moonraker/components/update_manager/git_deploy.py", line 1260, in _run_git_cmd_async
    raise self.server.error(
moonraker.utils.ServerError: Unable to repair loose objects, use hard recovery
2023-12-08 15:06:59,843 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2023-12-08 15:06:59,844 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2023-12-08 15:06:59,845 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2023-12-08 15:08:00,398 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 127.0.0.1
2023-12-08 15:08:00,410 [app.py:log_request()] - 101 GET /websocket (127.0.0.1) [_API_KEY_USER_] 0.76ms
2023-12-08 15:08:00,410 [websockets.py:open()] - Websocket Opened: ID: 140285712576048, Proxied: False, User Agent: , Host Name: 127.0.0.1
2023-12-08 15:08:34,680 [app.py:log_request()] - 101 GET /websocket (127.0.0.1) [_TRUSTED_USER_] 0.71ms
2023-12-08 15:08:34,680 [websockets.py:open()] - Websocket Opened: ID: 140285713010400, Proxied: False, User Agent: , Host Name: 0.0.0.0
2023-12-08 15:08:34,682 [websockets.py:_handle_identify()] - Websocket 140285713010400 Client Identified - Name: OctoEverywhere, Version: 2.7.6, Type: agent
2023-12-08 15:11:33,697 [machine.py:_find_public_ip()] - Detected Local IP: 192.168.1.223
2023-12-08 15:36:05,379 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 15:36:07,390 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 15:36:09,400 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 16:06:05,382 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 16:06:07,392 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 16:06:09,402 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 16:11:03,377 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 127.0.0.1
2023-12-08 16:36:05,381 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 16:36:07,394 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 16:36:09,405 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 17:06:05,382 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 17:06:07,392 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 17:06:09,402 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 17:36:05,384 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 17:36:07,395 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 17:36:09,405 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 18:06:05,387 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 18:06:07,398 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 18:06:09,407 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 18:36:05,386 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 18:36:07,396 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 18:36:09,406 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 19:06:05,388 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 19:06:07,400 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 19:06:09,410 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 19:36:05,391 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 19:36:07,401 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 19:36:09,412 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 20:06:05,394 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 20:06:07,404 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 20:06:09,415 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 20:36:05,391 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 20:36:07,401 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 20:36:09,411 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 21:06:05,394 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 21:06:07,402 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 21:06:09,410 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 21:36:05,396 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 21:36:07,406 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 21:36:09,415 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 22:06:05,396 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 22:06:07,408 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 22:06:09,420 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 22:36:05,397 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 22:36:07,406 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 22:36:09,416 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 23:06:05,395 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 23:06:07,404 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 23:06:09,417 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
2023-12-08 23:36:05,397 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'moonraker': [Errno -3] Temporary failure in name resolution
2023-12-08 23:36:07,407 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'klipper': [Errno -3] Temporary failure in name resolution
2023-12-08 23:36:09,417 [announcements.py:_fetch_moonlight()] - Failed to update subscription 'octoeverywhere': [Errno -3] Temporary failure in name resolution
